configs/RCCS-Fugaku-Fujitsu-A64FX-TofuD/spack.yaml

```diff
@@ -6,20 +6,31 @@
 spack:
   packages:
     default-compiler:
-      spack_spec: fj@{default_fj_version}
-    default-mpi:
-      spack_spec: fujitsu-mpi@{default_fj_version}%fj arch=linux-rhel8-a64fx
-    compiler-gcc:
+      #spack_spec: fj@{default_fj_version}
+      #spack_spec: clang@{default_llvm_version}
       spack_spec: gcc@{default_gnu_version}
+    default-mpi:
+      spack_spec: fujitsu-mpi@{default_fj_version} arch=linux-rhel8-a64fx
     compiler-fujitsu:
       spack_spec: fj@{default_fj_version}
+    compiler-clang:
+      spack_spec: clang@{default_llvm_version}
```

Comment:
(no change needed) This may be required until we invoke Ramble in a way that avoids having it look for additional compilers.